tools/checky/src/checks.rs

```diff
@@ -0,0 +1,15 @@
+use std::ops::Range;
+
+use crate::rust_file::RustFile;
+
+pub mod empty_lines;
+pub mod mod_use_order;
+
+pub type Report<'a> = ariadne::Report<'a, (&'a str, Range<usize>)>;
+
+pub fn check(file: &RustFile) -> Vec<Report> {
+    let mut reports = Vec::new();
+    reports.extend(empty_lines::check(file));
+    // reports.extend(mod_use_order::check(file));
```

Comment:
Why is it disabled? If it is dead code, we should remove it for now?

Reply:
We need to first decide on the way how we want to order that. I forgot what we decided in some Dev-Meeting. So maybe you need to decide it again. Then we can enable the check and fix all wrong mod-use-orders.